### PR TITLE
Connect2All

### PR DESCRIPTION
**Pull request by Joerio**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

File: certified-connectors/Connect2All/readme.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## GAC Connect2All
+GAC Connect2All integrates the PowerAutomate platform to Business Central and makes adaptations and the extensibility to connect virtual any Datasource to your business processes in Business Central possible. You do not need to be a Professional developer to seamlessly extend Business Central to interact to other business application and Datasources. GAC Connect2All delivers your business the scalability and flexibility to create smart integration solutions you need today and tomorrow.
```
Comment (dougelliott) [inline certified-connectors/Connect2All/readme.md]: >integrates the PowerAutomate platform to Business Central

Could this please be changed to say "integrates the Microsoft Power Platform to..."

File: certified-connectors/Connect2All/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "30118c48-c78c-40a1-84b7-80961849e93b",
```
Comment (dougelliott) [inline certified-connectors/Connect2All/apiProperties.json]: Please remove the client ID and replace it with a dummy value

File: certified-connectors/Connect2All/readme.md (hunk 1 above)
Comment (Joerio) [reply certified-connectors/Connect2All/readme.md]: I updated this text as suggested.

File: certified-connectors/Connect2All/apiProperties.json (hunk 2 above)
Comment (Joerio) [reply certified-connectors/Connect2All/apiProperties.json]: Removed the clientId and put in a dummy value.

**Pull request merged**